src/QueryLogger.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Muffin\Webservice;
+
+class QueryLogger extends \Cake\Database\Log\QueryLogger
```

Comment:
The reason I'm using the ```\Cake\Database\Log\QueryLogger``` is integration with DebugKit.  DebugKit integration just works when using the QueryLogger.